content/en/Integrations/webhooks.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Get Pentest Updates with Webhooks"
+linkTitle: "Webhooks"
+weight: 10
+description: >
+  Set up webhook notifications for your organization.
+---
+
+{{% pageinfo %}}
+Configure webhooks to subscribe to real-time updates for your pentests.
+{{% /pageinfo %}}
+
+With our API-based webhooks, you can set up an integration between your app and the Cobalt platform to get notifications for pentest events. We’ll send you updates for each event to your URL through an HTTP POST request.
+
+When you work with an API, you can become aware of new data in the following ways:
+
+- Repeatedly send requests to the same [API endpoint](/getting-started/glossary/#api-endpoint) to retrieve new information. This process is known as API polling.
+- Configure a webhook that automatically sends updates for new data to the specified URL.
+
+{{< mermaid align="left" theme="default" >}}
+sequenceDiagram
+  title Webhook for pentest events
+   Your App->>Cobalt Platform: Notify me of new pentest events
+   Cobalt Platform-->>Your App: Pentest created
+   Cobalt Platform-->>Your App: Finding created
+   Cobalt Platform-->>Your App: Asset updated
+   Cobalt Platform-->>Your App: Finding state changed
+{{< /mermaid >}}
+<br>
+
+You can configure webhooks through the API and in the [Cobalt UI](#configure-webhooks-in-the-ui).
+<!-- Provide a link to API docs or API use case. -->
+
+## Before You Start
+
+Before you start creating webhooks, complete the configuration in your app.
+
+1. Generate an **authentication token**. We need this token to authorize HTTP POST requests to your URL and connect to your app.
+1. Make sure that the **URL** where you want to receive notifications is valid and your services work properly.
+
+## Webhook Events
+
+When you set up a webhook, you subscribe to the following events related to your pentests.
+
+| Asset | Pentest | Finding |
+|---|---|---|
+| <li>Asset created</li><li>Asset updated</li><li>Asset deleted</li> | <li>Pentest created</li><li>[Pentest state](/penteststates/) changed</li><li>Pentest deleted</li> | <li>Finding created</li><li>Comment created for a finding</li><li>Finding state changed</li><li>Finding deleted</li>
+
+## Configure Webhooks in the UI
+
+Let’s configure webhooks in the Cobalt app.
+
+### Create a Webhook
+
+To create a webhook:
+
```

Comment:
Should we include a "Sign in to Cobalt" step?

Reply:
@mjang-cobalt Please see my previous comment and let me know if that works.

Reply:
You do now say "In the Cobalt app"... 
(or did I miss that earlier?)

Reply:
I made a commit to the file, I should have added a suggestion.